src/topicFactory.js

```diff
@@ -3,91 +3,100 @@ import qs from 'qs'
 import randomString from './util/randomString'
 const CHASQUI_URL = 'https://chasqui.uport.me/api/v1/topic/'
 
-function TopicFactory (isOnMobile, pollingInterval = 2000, chasquiUrl = CHASQUI_URL) {
```

Comment:
@zachferland For the closure to work it's pretty simple you just add `let stopBool` here after the function declaration.